R/ft_get_si.R

```diff
@@ -0,0 +1,240 @@
+#' Download supplementary materials from journals
+#'
+#' Download supplementary materials from papers. Put a call to this
+#' function where you would put a file-path - everything is cached by
+#' default, so you don't have to worry about multiple downloads in the
+#' same session.
+#' @param doi DOI of article. Note: if using ESA journal, this must be
+#' the ESA-specific article code (e.g., E092-201).
+#' @param si number of the supplment to be downloads (1, 2, 3, etc.),
+#' or (for ESA and Science journals) the name of the supplment (e.g.,
+#' "S1_data.csv").
+#' @param from Publisher of article. Optional, except for ESA journals
```

Comment:
it might be helpful to have more information here, e.g. a link for each publisher, could make a separate list in `@details`